rest-server/src/config/express/index.js

```diff
@@ -0,0 +1,34 @@
+import express from 'express';
+import parser from 'body-parser';
+import cors from 'cors';
+import helmet from 'helmet';
+
+import router from '../../routes/index';
+
+const middleWare = [
+  helmet(),
+  parser.json(),
+  parser.urlencoded({ extended: true }),
+  cors({
+    allowedHeaders: 'Content-Type, authorization',
+    methods: ['GET, POST, PUT, DELETE', 'OPTIONS'],
```

Comment:
Be careful here. You do not support OPTIONS.

Reply:
`OPTIONS` is required by the cors handler.